src/main/java/com/fabbroniko/scene/GameScene.java

```diff
@@ -1,10 +1,7 @@
 package com.fabbroniko.scene;
 
-import com.fabbroniko.environment.Background;
-import com.fabbroniko.environment.CollisionManager;
+import com.fabbroniko.environment.*;
```

Comment:
Make sure IntelliJ is set up to avoid including all classes in a package automatically